src/xxd/xxd.c

```diff
@@ -498,6 +517,11 @@ main(int argc, char *argv[])
   char *pp;
   char *varname = NULL;
   int addrlen = 9;
+  int color=0;
+
+#ifndef  __MVS__
```

Comment:
`__MVS__` is checking if the host system is z/OS, which I believe is not what you meant.